src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -231,21 +231,27 @@
       @end
       {@constructDefaults(service)}
 
-      var stub = gaxGrpc.createStub(
-          servicePath,
-          port,
-          grpcClient.{@service.getFullName},
-          {sslCreds: sslCreds});
-      var methods = [
-        @join method : context.getNonStreamingMethods(service) on {@","}.add(BREAK)
-          '{@methodName(method)}'
+      @let stubs = {@context.getStubs(service)}
+        @join stub : stubs on BREAK.add(BREAK)
+          var {@stub.name} = gaxGrpc.createStub(
+            servicePath,
+            port,
+            grpcClient.{@stub.grpcClientTypeName},
+            {sslCreds: sslCreds});
+          var {@stub.name}Methods = [
```

Comment:
If you want this to be in the spirit of MVVM, don't concatenate strings to form identifiers in the snippet. Make this variable name another attribute in the view (which not all languages will use).